### ISSUE TITLE
Method Sourcegraph\BuildServer\Server\TextDocument::implementation() does not exist

### ISSUE THREAD

**Issue by nesl247** [sourcegraph/sourcegraph#301]
- **Sourcegraph version:Sourcegraph.com
- **Platform information:** 

#### Steps to reproduce:

1. Find implementations on PHP interface method

#### Expected behavior:
For it to find implementations of a method

#### Actual behavior:
Error getting implementations: Request languageFeatures/$provideImplementation failed with message: Method Sourcegraph\BuildServer\Server\TextDocument::implementation() does not exist

**Comment by nesl247** [comment]
In that case a friendly warning should be shown some way in source graph.

**Comment by felixfbecker** [comment]
Yeah, we should not show the action buttons if the capability is not reported by the language server.
